fastlane/Fastfile

```diff
@@ -307,51 +337,71 @@ platform :android do
       aab_artifact_path = aab_artifact_path(app, version)
       UI.error("Unable to find a build artifact at #{aab_artifact_path}") unless File.exist? aab_artifact_path
 
-      track = case app
-              when APPS_AUTOMOTIVE
-                is_beta ? PLAY_STORE_TRACK_AUTOMOTIVE_BETA : PLAY_STORE_TRACK_AUTOMOTIVE_PRODUCTION
-              when APPS_WEAR
-                is_beta ? PLAY_STORE_TRACK_WEAR_BETA : PLAY_STORE_TRACK_WEAR_PRODUCTION
-              else
-                is_beta ? PLAY_STORE_TRACK_BETA : PLAY_STORE_TRACK_PRODUCTION
-              end
-
+      track = play_store_track(app: app, is_beta: is_beta)
       upload_to_play_store(
-        package_name: APP_PACKAGE_NAME,
+        **UPLOAD_TO_PLAY_STORE_COMMON_OPTIONS,
         aab: aab_artifact_path,
         track: track,
-        release_status: 'draft',
-        skip_upload_apk: true,
-        skip_upload_metadata: true,
-        skip_upload_changelogs: true,
-        skip_upload_images: true,
-        skip_upload_screenshots: true,
-        json_key: UPLOAD_TO_PLAY_STORE_JSON_KEY
+        release_status: 'inProgress',
+        rollout: is_beta ? '0.20' : '0.01' # Start at 20% for betas, 1% for Production
```

Comment:
After discussing with @MiSikora in DM, I'll revert this change of auto-submitting the builds for review—especially because PocketCasts Android does not use Managed Publishing in the Play Store Console.
We'll revert back to submitting them initially as draft to let the Release Manager test the APKs before calling `update_rollouts` only once the tests of the 3 APKs are satisfactory.